module-apps/application-music-player/windows/MusicPlayerAllSongsWindow.cpp

```diff
@@ -63,16 +57,10 @@ namespace gui
         setFocusItem(songsList);
     }
 
-    void MusicPlayerAllSongsWindow::destroyInterface()
-    {
-        erase();
-    }
-
     void MusicPlayerAllSongsWindow::onBeforeShow([[maybe_unused]] ShowMode mode, [[maybe_unused]] SwitchData *data)
     {
         presenter->attach(this);
         presenter->createData();
-        songsList->rebuildList(listview::RebuildType::OnPageElement);
```

Comment:
did you create bug to fix that behaviour we talked about ? Refocusing list on currently played track ?

Reply:
https://appnroll.atlassian.net/browse/EGD-7983